src/hoc/PrivateRoute.tsx

```diff
@@ -11,11 +11,11 @@ interface IProps {
 export function PrivateRoute({ el }: IProps) {
   const [offline] = useGlobal('offline');
   const { pathname } = useLocation();
-  const navigate = useNavigate();
+  const navigate = useMyNavigate();
   const { isAuthenticated } = useContext(TokenContext).state;
 
   if (typeof pathname === 'string' && pathname !== '/loading')
-    localStorage.setItem(localUserKey(LocalKey.deeplink), pathname);
+    localStorage.setItem(LocalKey.deeplink, pathname);
```

Comment:
Shouldn't we have a separate deep link for each user? It is not critical if we can't do it since most computers will have only one user anyway.

Reply:
No, we don't know who the user is when we deeplink.